packages/core/src/lib/tooltip/tooltip-container.svelte

```diff
@@ -34,7 +34,11 @@
 <script lang="ts">
 import { provideTooltipContext } from './tooltip-styles';
 
-const { id } = provideTooltipContext();
+export let hoverDelayMs: number | undefined = undefined;
```

Comment:
Given that tooltip style calculation is already async, I'm not sure we lose out on anything if we default to `0` rather than `undefined` and accept that we'll sometimes to `setTimeout(..., 0)`.

Alternatively, should we be setting a default non-zero timeout on this? I think that might be the design intent across the board

Reply:
Will set the default to `0` rather than `undefined` - I spoke with Nat and a delay is only sometimes used. For the most part, tooltips should be instant.